providers/implementations/kdfs/sskdf.c

```diff
@@ -368,6 +375,24 @@ static size_t sskdf_size(KDF_SSKDF *ctx)
     return (len <= 0) ? 0 : (size_t)len;
 }
 
+#ifdef FIPS_MODULE
+static int fips_sskdf_key_check_passed(KDF_SSKDF *ctx)
+{
+    OSSL_LIB_CTX *libctx = PROV_LIBCTX_OF(ctx->provctx);
+    int key_approved = ossl_kdf_check_key_size(ctx->secret_len * 8);
```

Comment:
might be better to shift the *8 into the function?

Reply:
Updated. Make the `ossl_kdf_check_key_size` function accept key length in bytes.